docs/semgrep-ci/manual-ci-setup.md

```diff
@@ -0,0 +1,507 @@
+* The value of the `SEMGREP_RULES` environment variable.
+
+The `SEMGREP_RULES` environment variable accepts a list of local and remote rules and rulesets to run. The `SEMGREP_RULES` list is delimited by a space (` `) if the variable is exported from a shell command or script block. For example, see the following BitBucket Pipeline snippet:
+
+```yaml
+# ...
+  script:
+    - export SEMGREP_RULES="p/nginx p/ci no-exec.yml" 
+    - semgrep ci
+# ...
+```
+
+The line defining `SEMGREP_RULES` defines three difference sources, delimited by a space:
+
+```
+- export SEMGREP_RULES="p/nginx p/ci no-exec.yml" 
+```
+
+The example references two rulesets from Semgrep Registry (`p/nginx` and `p/ci`)and a rule available in the repository (`no-exec.yml`).
+
+If the `SEMGREP_RULES` environment variable is defined from a YAML block, the list of rules and rulesets to run is delimited by a newline. See the following example of a GitLab CI/CD snippet:
+```YAML
+# ...
+variables:
+  SEMGREP_RULES: >-
+    p/nginx
+    p/ci
+    no-exec.yml
+# ...
+```
+
+#### Writing your own rules
+
+Write custom rules to enforce your team's coding standards and security practices. Rules can be forked from existing community-written rules.
+
+See [Writing rules](https://semgrep.dev/docs/writing-rules/overview/) to learn how to write custom rules.
+
+### Ignoring files
+
+By default `semgrep ci` skips files and directories such as `tests/`, `node_modules/`, and `vendor/`. It uses the default `.semgrepignore` file which can be found in the [Semgrep GitHub repository](https://github.com/returntocorp/semgrep/blob/develop/.semgrepignore). This default is used when no explicit `.semgrepignore` file is found in the root of your repository.
+
+You can copy and commit the default `.semgrepignore` file to the **root of your repository** and extend it with your own entries or write one from scratch. If Semgrep detects a `.semgrepignore` file within your repository, it won't append entries from the default `.semgrepignore` file.
+
+For a complete example, see the [.semgrepignore file on Semgrep’s source code](https://github.com/returntocorp/semgrep/blob/develop/.semgrepignore).
+
+:::caution
+`.semgrepignore` is only used by Semgrep CI and the Semgrep command line tool. Integrations such as [GitLab's Semgrep SAST Analyzer](https://gitlab.com/gitlab-org/security-products/analyzers/semgrep) do not use it.
+:::
+
+For information on ignoring individual findings in code, see the [Ignoring findings page](https://semgrep.dev/docs/ignoring-findings/).
+
+### Saving or exporting findings to a file
+
+To save or export findings, you can pass file format options and send the formatted findings to a file.
+
+For example, to save to a JSON file:
+
+`semgrep ci --json --output findings.json`
+
+You can also use output redirection (`>`):
+
+`semgrep ci --sarif > findings.sarif`
+
+Refer to the [CLI reference](/cli-reference) for output formats.
+
+## Migrating to Semgrep App from a manual CI setup
```

Comment:
I see, the section I recommended above is here - maybe we can link here from the Limitations section above? Just an idea.

Reply:
I prefer that we link to the actual Running Semgrep CI in Semgrep App doc, once it's available.